testbed/testbed/receivers.go

```diff
@@ -60,7 +60,7 @@ type DataReceiverBase struct {
 	Port int
 }
 
-const DefaultHost = "localhost"
+const DefaultHost = "127.0.0.1"
```

Comment:
this line was causing an error with the fluentbit log test since `localhost` could not be resolved on the github actions runners.